test/models/project_test.rb

```diff
@@ -20,7 +20,25 @@ def setup
   test '#contains_url?' do
     assert Project.new.send :contains_url?, 'https://www.example.org'
     assert Project.new.send :contains_url?, 'http://www.example.org'
+    assert Project.new.send :contains_url?,
+                            'See also <http://x.org>.'
+    assert Project.new.send :contains_url?,
+                            'See also <http://x.org>.'
     refute Project.new.send :contains_url?, 'mailto://mail@example.org'
     refute Project.new.send :contains_url?, 'abc'
+    refute Project.new.send :contains_url?,
+                            'See also http://x for more information.'
+    refute Project.new.send :contains_url?, 'www.google.com'
```

Comment:
`.send` is only necessary for testing private methods. Could you please change all of these to the form:
```ruby
refute Project.new.contains_url? 'mailto://mail@example.org'
```